myvar.h

```diff
@@ -40,7 +40,7 @@ extern struct MyVarList MyVars; ///< List of all the user's custom config variab
 
 void        myvar_del(const char *var);
 const char *myvar_get(const char *var);
-void        myvar_set(const char *var, const char *val);
+void        myvar_set(const char *var, const char *val, bool append);
```

Comment:
clang-format wants to change the layout not only of my changed line, but also the neighboring lines.  Should I prefer prevailing style, or allow the changes?

Reply:
For the header files, don't worry.

We mandate `clang-format` for the source, but humans benefit from some extra whitespace in the headers.